components/brex/actions/list-transaction-for-selected-cash-account/common.mjs

```diff
@@ -0,0 +1,51 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  props: {
+    postedAtStart: {
+      type: "string",
+      label: "Posted At Start",
+      description: "Shows only transactions with a posted_at_date on or after this date-time. This parameter is the date-time notation as defined by [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). Example: `2022-12-12T23:59:59.999`",
+      optional: true,
+    },
+    max: {
+      type: "integer",
+      label: "Max",
+      description: "The maximum amount of registered that will be fetched. Defaults to `500`.",
+      optional: true,
+    },
```

Comment:
Since `postedAtStart` and `max` are used in both `list-transaction-for-selected-cash-account` and `list-transaction-for list-transactions-for-primary-card-account`, they should probably be listed in the app file under propDefinitions.

I think if the `max` is reached, no additional pages are retrieved, but it still might emit more transactions than the `max`. For instance, my account showed a total of 3 transactions. If I put in 2 as the `max`, I still see all 3 transactions returned because they're all on the same "page".

Is the word "registered" correct in the description for `max`. Should it be "transactions"?